### PR TITLE
Covered topic flags with tests

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -483,6 +483,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "fd11e2094673d9a89cee7b47b626c2c18c7dd80b6e86835d88a2d8d28e5e9fa7"
+  inputs-digest = "2bdb7a811f635e7d86992a28dfa158f917167dd443bd5766945492c52bbd6534"
```
Comment (mandrigin) [inline Gopkg.lock]: 🤔

**Review by mandrigin — CHANGES_REQUESTED** [review]
the changes here look very similar to a peer pool. what is the difference?

**Comment by dshulyak** [comment]
@mandrigin i just want to land more of the same code :)

i need to rebase. the difference is in cmd/topics/ module. there are new tests

**Comment by mandrigin** [comment]
@dshulyak ah, sure! let's land the peer pool first (one conflict left) and then that'd be much easier to review this one :)

**Comment by dshulyak** [comment]
@mandrigin rebased

**Review by mandrigin — APPROVED** [review]
Looks good!
I would also have a README.md in the `topics` package with a few examples and maybe a link to the unit-tests for more deep details.

**Review by adambabik — APPROVED** [review]

**Pull request merged**